addons/website/static/lib/summernote/src/js/Renderer.js

```diff
@@ -409,7 +409,8 @@ define([
         var content = '<div class="btn-group">' + fullButton + halfButton + quarterButton + '</div>' +
                       '<div class="btn-group">' + leftButton + rightButton + justifyButton + '</div>' +
                       '<div class="btn-group">' + roundedButton + circleButton + thumbnailButton + noneButton + '</div>' +
-                      '<div class="btn-group">' + removeButton + '</div>';
+                      '<div class="btn-group">' + removeButton + '</div>'+
+                      '<div class="note-color btn-group hidden" style="display:none;">'+tplButtonInfo.color(lang, options) + '</div>';
```

Comment:
Unauthorized to change summernote
Otherwise I could not follow up and work with the official library